LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
```

Comment:
Does this mean that this license file will show up in the documentation? Not sure if this is what we want?